array-map/main.js

```diff
@@ -0,0 +1,21 @@
+const numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12];
+
+const languages = [
+  'Hypertext Markup Language',
+  'Cascading Style Sheets',
+  'ECMAScript',
+  'PHP Hypertext Preprocessor',
+  'Structured Query Language'
+];
+
+const doubled = numbers.map(x => x * 2);
+console.log(doubled);
+
+const prices = numbers.map(x => '$' + x);
+console.log(prices);
```

Comment:
Please also make sure the price strings have exactly two decimal places:

---

![image](https://user-images.githubusercontent.com/16143771/206788406-15336788-5580-4ad4-b90a-a26fc5456799.png)